backend/internal/repository/germline_cnv_occurrences.go

```diff
@@ -0,0 +1,91 @@
+package repository
+
+import (
+	"fmt"
+	"github.com/radiant-network/radiant-api/internal/utils"
+	"log"
+
+	"github.com/Goldziher/go-utils/sliceutils"
+	"github.com/radiant-network/radiant-api/internal/types"
+	"gorm.io/gorm"
+)
+
+type GermlineCNVOccurrence = types.GermlineCNVOccurrence
+
+type GermlineCNVOccurrencesRepository struct {
+	db *gorm.DB
+}
+
+type GermlineCNVOccurrencesDAO interface {
+	GetOccurrences(seqId int, userFilter types.ListQuery) ([]GermlineCNVOccurrence, error)
+	CountOccurrences(seqId int, userFilter types.CountQuery) (int64, error)
+}
+
+func NewGermlineCNVOccurrencesRepository(db *gorm.DB) *GermlineCNVOccurrencesRepository {
+	if db == nil {
+		log.Print("GermlineCNVOccurrencesRepository: db is nil")
+		return nil
+	}
+	return &GermlineCNVOccurrencesRepository{db: db}
+}
+
+func (r *GermlineCNVOccurrencesRepository) GetOccurrences(seqId int, userQuery types.ListQuery) ([]GermlineCNVOccurrence, error) {
```

Comment:
Since the logic is exactly the same as in SNV, maybe refactor it with a generic typing to avoid duplication